airgun/views/os.py

```diff
@@ -10,6 +10,59 @@
 )
 
 
+class TemplatesList(View):
+    """List of templates for specific operating system. It can have dynamic
+    number of templates per different OS types
+
+    Example html representation::
+
+        <label ... for="provisioning_template_id">PXELinux template *</label>
+        <div...>
+            <div class="..." id="s2id_operatingsystem_os_default_templates...">
+                <a>
+                    <span>Kickstart default PXELinux</span>
+        ...
+        <label ... for="provisioning_template_id">PXEGrub template *</label>
+        <div...>
+            <div class="..." id="s2id_operatingsystem_os_default_templates...">
+                 <a>
+                     <span>Kickstart default PXEGrub2</span>
+
+    """
+    SELECT = "//label[@for='provisioning_template_id'][contains(.,'%s')]" \
+             "/following-sibling::div/div[contains(@id, 'default_templates')]"
+    TITLES = "//label[@for='provisioning_template_id']"
+
+    @property
+    def selects(self):
+        """Get dictionary of currently assigned templates for OS"""
+        selects = {}
+        for title in self.browser.elements(
+                self.TITLES, parent=View, check_visibility=True):
```

Comment:
parent=View? did you mean parent=self or no parent at all, since you use absolute locators?

Reply:
parent is not self, as self is not View